<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrismViewScalingWidget</class>
 <widget class="QWidget" name="PrismViewScalingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change View Scale</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Full Bounds</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Threshold Bounds</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Custom Scale</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Y Axis</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Z AXis</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="UseXCustomScale">
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="XCustomMin">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="XCustomMax">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="4">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="UseYCustomScale">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="YCustomMin">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="YCustomMax">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="4">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QRadioButton" name="UseZCustomScale">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ZCustomMin">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ZCustomMax">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>X Axis</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QRadioButton" name="UseZThresholdBounds">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QRadioButton" name="UseYThresholdBounds">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QRadioButton" name="UseXThresholdBounds">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QRadioButton" name="UseZFullBounds">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QRadioButton" name="UseYFullBounds">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QRadioButton" name="UseXFullBounds">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>UseXFullBounds</tabstop>
  <tabstop>UseXThresholdBounds</tabstop>
  <tabstop>UseXCustomScale</tabstop>
  <tabstop>XCustomMin</tabstop>
  <tabstop>XCustomMax</tabstop>
  <tabstop>UseYFullBounds</tabstop>
  <tabstop>UseYThresholdBounds</tabstop>
  <tabstop>UseYCustomScale</tabstop>
  <tabstop>YCustomMin</tabstop>
  <tabstop>YCustomMax</tabstop>
  <tabstop>UseZFullBounds</tabstop>
  <tabstop>UseZThresholdBounds</tabstop>
  <tabstop>UseZCustomScale</tabstop>
  <tabstop>ZCustomMin</tabstop>
  <tabstop>ZCustomMax</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>UseXCustomScale</sender>
   <signal>toggled(bool)</signal>
   <receiver>XCustomMax</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UseYCustomScale</sender>
   <signal>toggled(bool)</signal>
   <receiver>YCustomMax</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UseZCustomScale</sender>
   <signal>toggled(bool)</signal>
   <receiver>ZCustomMax</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UseXCustomScale</sender>
   <signal>toggled(bool)</signal>
   <receiver>XCustomMin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UseYCustomScale</sender>
   <signal>toggled(bool)</signal>
   <receiver>YCustomMin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UseZCustomScale</sender>
   <signal>toggled(bool)</signal>
   <receiver>ZCustomMin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
